README.rst

```diff
@@ -223,28 +223,26 @@ Add direct support to scripts for a little more configurability:
     def get_main_parser():
         parser = argparse.ArgumentParser(prog="pathcomplete")
         parser.add_argument(
-            "-s", "--print-completion-shell", choices=["bash", "zsh"]
-        )
-        parser.add_argument(
-            "--file",
-            choices=shtab.Optional.FILE,  # file tab completion, can be blank
-        )
-        parser.add_argument(
-            "--dir",
-            choices=shtab.Required.DIRECTORY,  # directory tab completion
-            default=".",
+            "-s",
+            "--print-completion-shell",
+            choices=["bash", "zsh"],
+            help="prints completion script",
         )
+        # "file" & "directory" tab complete
+        parser.add_argument("file", nargs="?").complete = "file"
+        parser.add_argument("--dir", default=".").complete = "directory"
```

Comment:
This made me stop and think wheather or not this is a regex that should match by this name or what it was before :) shtab.FILE/DIR felt nicer, tbh.

Reply:
sure; added